CONTRIBUTING.md

```diff
@@ -83,13 +83,13 @@ When a new component is being created, work with the design team to document des
 #### Consider the Following Before Adding a New Component
 
 **Is it likely this component be used in more than 1 app?**
-For example, a component for a card that may contain any content would be a good candidate for ui-kit. A card that specifically renders data specific to an object in an napp is not.
+For example, a component for a card that may contain any content would be a good candidate for ui-kit. A card that specifically renders data specific to an object in an app is not.
```

Comment:
I could have used you in college 😂

Reply:
I took a copy editing course with my Journalism minor that will forever haunt me. 😆